docs/tour/interactive.md

```diff
@@ -41,6 +41,9 @@ tooltips.
 
 Links are like tooltips, except you click to go to an external link.
 
+When the link is a URI that ends with a fragment, e.g., `https://example.com/page#fragment`,
+it must be written between double-quotes to prevent the fragment from being discarded.
```

Comment:
Let's make this an info block:

```
:::info
When the link contains the `#` character as part of a URI fragment, e.g....., remember that it will be treated as a comment if unquoted and unescaped.
```

That's more accurate than needs double-quotes.
https://play.d2lang.com/?script=qtDLyczLtlLIKCkpKLbS10-tSMwtyEnVS87P1S9ITE-NUU4rSkzPTc0r4aqEKlUnrFadqwqqWImwYiUuQAAAAP__&

Reply:
Updated.